CONTRIBUTING.md

```diff
@@ -165,4 +166,18 @@ Please keep in mind that the team may **suspend or reject** pull requests. There
 -   Poor quality code.
 -   Other [development priorities](https://testcafe.io/402949/roadmap) may take precedence.
 
-Please also note that we do not merge pull requests until the changes are documented.
+We merge pull requests after the changes are documented. If you want to document a new capability, write a comment with your description to a pull request. Please, follow the [guidelines](#guidelines) during the writing. 
+
+## Contribute to Documentation
+
+Users can't submit documentation pull requests because TestCafe stores [documentation](https://testcafe.io/documentation) in the private repository. To notify the team of an inaccuracy in the docs, create a [Github issue](https://github.com/DevExpress/testcafe/issues/new) with the `AREA: docs` label.
+
+#### Guidelines
+
+TestCafe team adheres to a set of writing guidelines and practices that make our docs easy to read. When you submit a comment about a new capability or a new issue of an inaccuracy, please, follow the main rules:
+
+-   You don't use gerunds, passive voice and past tense
+-   Every sentence expresses a single idea
+-   You don't needlessly repeat the same piece of information
```

Comment:
- these aren't rules
- the list isn't uniform
- this isn't a great selection, we probably need a couple more (maybe it's a subject for a later discussion)

```suggestion
-   Avoid gerunds, passive voice and past tense.
-   Keep your sentences simple. Make sure that each sentence only expresses a single idea.
-   Do not needlessly repeat the same piece of information.
```